src/runtime/eng/pipelines/coreclr/templates/jit-replay-pipeline.yml

```diff
@@ -43,7 +43,7 @@ extends:
                   archiveExtension: $(archiveExtension)
                   artifactName: BuildArtifacts_$(osGroup)$(osSubgroup)_$(archType)_$(_BuildConfig)
                   displayName: Build Assets
-            condition: not(eq(stageDependencies.EvaluatePaths.evaluate_paths.outputs['SetPathVars_jiteeversionguid.containsChange'], true))
+            condition: and(succeeded(), not(eq(stageDependencies.EvaluatePaths.evaluate_paths.outputs['SetPathVars_jiteeversionguid.containsChange'], true)))
```

Comment:
I'm not sure this is correct, would need to follow the template expansion. I'd assume it's a no-op since this job probably doesn't depend on anything

Reply:
I'll remove the changes made to jobs and templates - I think they were lower confidence anyhow.